src/main/java/graphql/validation/ValidationUtil.java

```diff
@@ -156,9 +160,16 @@ boolean isValidLiteralValue(Value<?> value, GraphQLInputObjectType type, GraphQL
             }
 
         }
+        if (type.hasAppliedDirective(Directives.OneOfDirective.getName())) {
+            if (objectFieldMap.keySet() .size() != 1) {
```

Comment:
```suggestion
            if (objectFieldMap.keySet().size() != 1) {
```
Hyper nit: there's a extra space hanging about